test/scripts/create_payment.js

```diff
@@ -6,6 +6,47 @@ import { getKvEntryWithFallback } from './get_task_from_kv'
 import { KEY_VALUE_KEYS } from '../../src/const'
 import { balanceHelper, paymentHelper } from '../helpers'
 
+export async function constructPaymentWithSource (fromAccount, toAccount, assetCode, amount) {
+  return {
+    source: fromAccount.accountId(),
+    ...await constructPayment(fromAccount, toAccount, assetCode, amount)
+  }
+}
+
+export async function constructPayment (fromAccount, toAccount, assetCode, amount) {
```

Comment:
Tryna create configurable payment constructor here. I noticed that up to present days there were no ways to create configurable payment, only hard-coded version for tests was available